src/components/Home/Alumni/Alumni.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import './alumni.css';
+import { companyListAlumni, universityList } from '../../data';
+
+const Alumni = () => {
+    const [companies, setCompanies] = useState(companyListAlumni);
+    const [universities, setUniversities] = useState(universityList);
+
+    const renderLink = (array) => {
+        return array.map((item, index) => (
+            <span key={item.name}>
+                <span className="alumni__highlight">
+                    {item.name}
+                </span>
+                {(index < array.length - 1 && array.length > 2) ? ', ' : ''}
+                {(index < array.length - 1 && array.length === 2) ? ' ' : ''}
+                {(index === array.length - 2) ? 'and ' : ''}
+            </span>
+        ));
+    }
```

Comment:
The content does not contain any link. Consider changing the method name or add the links to each item.

Reply:
I'll add the link. However, feel free to check it if you have free time since I'm not sure whether the link I included is suitable or not (I chose the company career link).